library/src/main/java/com/airbnb/android/airmapview/WebViewMapFragment.java

```diff
@@ -156,6 +158,15 @@ public void setOnCameraChangeListener(OnCameraChangeListener listener) {
     onCameraChangeListener = listener;
   }
 
+  @Override
+  public void setOnMapInitialisedListener(OnMapInitializedListener listener) {
+    this.onMapInitializedListener = listener;
+    if (loaded) {
+      onMapInitializedListener.onMapInitialized();
```

Comment:
why is this needed? it's not actually initialized at this point -- if someone sets a new initialized listener far after it's initialized, we don't want to re-run any initialization code the developer may have added. we should remove this